src/box/lua/tuple.c

```diff
@@ -823,6 +924,19 @@ lbox_tuple_format_new(struct lua_State *L)
 	return lbox_push_tuple_format(L, format);
 }
 
+static const struct luaL_Reg lbox_tuple_format_meta[] = {
+	{"__gc", lbox_tuple_format_gc},
+	{"__ipairs", lbox_tuple_format_pairs},
+	{"__pairs", lbox_tuple_format_pairs},
+	{"__serialize", lbox_tuple_format_serialize},
+	{NULL, NULL}
+};
+
+static const struct luaL_Reg lbox_tuple_formatlib[] = {
+	{"_new", lbox_tuple_format_new},
```

Comment:
Should be defined in `box.internal.tuple` instead of using `_` prefix.

Reply:
Fixed. Also fixed same issue with `box.tuple.format._new`.